pkg/helpers.go

```diff
@@ -474,15 +476,79 @@ func ReplaceTerraformS3Backend() error {
 		return err
 	}
 
+	// update GitHub Terraform content
```

Comment:
will this break during a Github installation?

Reply:
good catch!! tks! https://github.com/kubefirst/kubefirst/pull/713